{"name": "test_failed_new_users[payload2]", "status": "passed", "steps": [{"name": "Creating user with invalid credentials", "status": "passed", "start": 1742319239364, "stop": 1742319239372}, {"name": "Checking response status is 400", "status": "passed", "start": 1742319239373, "stop": 1742319239373}], "parameters": [{"name": "payload", "value": "({'username': 'problem_user1', 'password': 'secret_sauce'}, True)"}], "start": 1742319239364, "stop": 1742319239373, "uuid": "063efa59-d117-4ae8-871d-fc1e804c94bc", "historyId": "0392b2392117a83931684fa6d373cb7d", "testCaseId": "737231c88290cb7af71acf5305d104b9", "fullName": "tests.api.tests.test_full_cycle#test_failed_new_users", "labels": [{"name": "feature", "value": "Users"}, {"name": "severity", "value": "normal"}, {"name": "story", "value": "Create invalid users"}, {"name": "parentSuite", "value": "tests.api.tests"}, {"name": "suite", "value": "test_full_cycle"}, {"name": "host", "value": "562b1fef8d81"}, {"name": "thread", "value": "19-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.api.tests.test_full_cycle"}]}